packages/engine/src/renderer/functions/configureEffectComposer.ts

```diff
@@ -81,14 +85,20 @@ export const configureEffectComposer = (
 
   const effects: any[] = []
 
-  const smaaEffect = new SMAAEffect()
+  const smaaEffect = new SMAAEffect({
+    preset: SMAAPreset.HIGH,
+    edgeDetectionMode: EdgeDetectionMode.COLOR
+  })
   composer.SMAAEffect = smaaEffect
 
   const outlineEffect = new OutlineEffect(scene, camera, getState(HighlightState))
+  outlineEffect.selectionLayer = ObjectLayers.HighlightEffect
```

Comment:
Had to change the layer of the outline because the default is 10 and we use that for transform Gizmo. @JT00y you'll have to likely do the same when using outline component